app/views/dashboards/show.html.erb

```diff
@@ -251,6 +251,25 @@
   <a href="<%= profile_path %>">
   <%= image_tag 'profile_icon.png', alt: 'Profile Icon', class: 'profile-icon' %>
   </a>
+    <div class="container my-5 pt-5">
+    <div class="row justify-content-center">
+        <div class="col-md-5 mb-3">
+            <%= link_to 'https://www.groupme.com/', target: "_blank" do %>

```

Comment:
Links will need to be replaced with specific links for the org. Functionality works great though from my tests. Emailing the org about the links right now.